### ISSUE TITLE
it cause hang up on ubuntu 18.02 box when i run "sudo wifiphisher" after the installation then i need to force shutdown to be able to use my box again.

### ISSUE THREAD

**Issue by robertcanare** [wifiphisher/wifiphisher#929]
Before opening a new issue, please make sure that what you are about to report has not been reported already. If not, use the following template for reporting.

**Version**: Use `git show | grep commit` command and paste here the output.

**Description**: Give a more analytical description here. The output of the `iw list` and `iwconfig` commands may be helpful.

**Script output**: Paste the output of the tool, including any stack traces.

**Comment by blackHatMonkey** [comment]
@robertcanare Thanks for opening a bug report :smile:. How did you install `wifiphisher`? `git`, `pip` or something else?

**Comment by robertcanare** [comment]
@blackHatMonkey  yup! once i run it its totaly hang up my physical machine can't use it anymore until i reboot it. i run wifiphisher before on same hardware but with kali on it,

**Comment by usama7628674** [comment]
@robertcanare Please post the log of `dmesg`
